From: Elias Werberich <elias@werberich.de>
Date: Tue, 6 Jun 2017 22:23:10 +0200
Subject: [PATCH] src: Remove packet warnings

It may happen that some packets could not be sent because of a full
buffer or other connectivity issues. This is normal and handeld by other
protocols. Warnings are removed because they may fill logs quickly.

Signed-off-by: Elias Werberich <elias@werberich.de>
---
 mainloop.ic | 12 +++---------
 1 file changed, 3 insertions(+), 9 deletions(-)

diff --git a/mainloop.ic b/mainloop.ic
index 2de5a38..90045ec 100644
--- mainloop.ic
+++ mainloop.ic
@@ -117,9 +117,7 @@ static void mainLoop() {
 				// output packets
 				while((sockdata_len = (p2psecOutputPacket(g_p2psec, sockdata_buf, 4096, new_peeraddr.addr))) > 0) {
 					sockdata_lastlen = sockdata_len;
-					if(!(ioWriteGroup(&iostate, IOGRP_SOCKET, sockdata_buf, sockdata_len, &new_peeraddr) > 0)) {
-						logWarning("could not send packet!");
-					}
+					ioWriteGroup(&iostate, IOGRP_SOCKET, sockdata_buf, sockdata_len, &new_peeraddr);
 				}
 			}
 			ioGetClear(&iostate, fd);
@@ -200,9 +198,7 @@ static void mainLoop() {
 						// output packets
 						while((sockdata_len = (p2psecOutputPacket(g_p2psec, sockdata_buf, 4096, new_peeraddr.addr))) > 0) {
 							sockdata_lastlen = sockdata_len;
-							if(!(ioWriteGroup(&iostate, IOGRP_SOCKET, sockdata_buf, sockdata_len, &new_peeraddr) > 0)) {
-								logWarning("could not send packet!");
-							}
+							ioWriteGroup(&iostate, IOGRP_SOCKET, sockdata_buf, sockdata_len, &new_peeraddr);
 						}
 					}
 				}
@@ -215,9 +211,7 @@ static void mainLoop() {
 		// output packets
 		while((sockdata_len = (p2psecOutputPacket(g_p2psec, sockdata_buf, 4096, new_peeraddr.addr))) > 0) {
 			sockdata_lastlen = sockdata_len;
-			if(!(ioWriteGroup(&iostate, IOGRP_SOCKET, sockdata_buf, sockdata_len, &new_peeraddr) > 0)) {
-				logWarning("could not send packet!");
-			}
+			ioWriteGroup(&iostate, IOGRP_SOCKET, sockdata_buf, sockdata_len, &new_peeraddr);
 		}
 
 		// show status
-- 
2.13.0
